exchange/exchange.go

```diff
@@ -742,12 +736,12 @@ func (e *exchange) getAllBids(
 		}, chBids)
 		go bidderRunner(bidder, conversions)
 	}
-	var fledge *openrtb_ext.Fledge
 
 	// Wait for the bidders to do their thing
 	for i := 0; i < len(bidderRequests); i++ {
 		brw := <-chBids
 
+		auctionResponseExtraInfo.bidderResponseStartTime = brw.bidderResponseStartTime
```

Comment:
The last recorded start time is asigned here but, we don't do anything with `auctionResponseExtraInfo.bidderResponseStartTime` downstream. Are we going to log the metrics in a separate PR?

Reply:
Yes, Hans and I asked about this earlier :)

Reply:
we will be having separate PR to record `bidderResponseStartTime`